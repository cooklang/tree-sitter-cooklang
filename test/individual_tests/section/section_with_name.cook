== Filling ==
Add @cheese{100%g}.