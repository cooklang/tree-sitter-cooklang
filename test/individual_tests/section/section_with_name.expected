(recipe 
  (section
    header: (section_header name: )
    content: (step (ingredient (name) (amount (quantity) (units))))))